test/SpeedJumpIrmTest.sol

```diff
@@ -76,22 +76,22 @@ contract AdaptativeCurveIrmTest is Test {
 
         market.totalBorrowAssets = 1 ether;
         market.totalSupplyAssets = 1 ether;
-        market.lastUpdate = uint128(block.timestamp - 30 days);
+        market.lastUpdate = uint128(block.timestamp - 5 days);
```

Comment:
Wouldn't it be actually interesting to keep the same duration and see the difference?

Reply:
it was a little bit too big, but I can roll back